src/components/Radio/Radio.module.css

```diff
@@ -9,3 +9,88 @@
   display: flex;
   gap: var(--eds-size-1);
 }
+
+/**
+ * Wraps the visually hidden radio input element and the visible sibling svg element.
+ */
+.input__wrapper {
+  position: relative;
+  /* Centers the radio icon in the wrapper. */
+  display: inline-flex;
+  align-items: center;
+  /* Aligns the radio with the first line of the label. */
+  align-self: flex-start;
+}
+/**
+ * The visually hidden input element for the radio. The visual radio is provided by an svg element.
+ */
+.radio__input {
+  /* Removes default margins placed by browser for radioes. */
+  margin: 0;
+  /* Remove the radio from the page flow, positioning it on top of the SVG. */
+  position: absolute;
+  left: 0.25rem;
+  /* Set same dimensions as the RadioSvg element. */
+  height: var(--eds-size-2);
+  width: var(--eds-size-2);
+  /**
+   * Hide the input element visually.
+   * This ensures the radio is still "physically" present so that all users,
+   * especially on touch screen readers, still interact with the real radio element
+   * where it would naturally be present.
+   */
+  opacity: 0;
+}
+
+/**
+ * The visible radio svg icon element.
+ */
+.radio__icon {
+  color: var(--eds-theme-color-radio-brand-background);
+  /* Creates space for the border so that there's no jitter when the focus border is visible. */
+  border: var(--eds-border-width-sm) solid transparent;
+}
+.radio__input:focus-visible + .radio__icon {
+  border: var(--eds-border-width-sm) solid var(--eds-theme-color-focus-ring);
+  border-radius: var(--eds-border-radius-full);
+}
+
+@supports not selector(:focus-visible) {
+  .radio__input:focus + .radio__icon {
+    border: var(--eds-border-width-sm) solid var(--eds-theme-color-focus-ring);
+    border-radius: var(--eds-border-radius-full);
+  }
+}
+
+/**
+ * The disabled variant of the visible radio svg icon.
+ */
+.radio__icon--disabled {
+  color: var(--eds-theme-color-icon-disabled);
+}
+
+/**
+ * The disabled status of the visually hidden input element.
+ */
+.radio__input:disabled {
+  /* Needed since the input element overlays the visible svg icon for user input and cursor. */
+  cursor: not-allowed;
+}
+
+/**
+ * Text that labels a radio input.
+ */
+.radio__label {
+  position: relative;
+}
+
+/**
+ * Radio label size variants.
+ * Used for centering the label with the radio input button.
+ */
+.radio__label--md {
+  top: 0.125rem;
+}
+.radio__label--lg {
+  top: 0.0625rem;
+}
```

Comment:
Not a fan of this, but again this was just brought over :)

Reply:
0.125rem corresponds to `--eds-size-quarter` which does exist, but `--eds-size-eighth` or what have you does not...and probably shouldn't. the sizes tokens are kinda weird. I wonder if there are other cases of such a small relative spacing that would indicate we should add this token?

Reply:
![image](https://github.com/chanzuckerberg/edu-design-system/assets/86632227/b1c6e05f-a956-4a73-9c93-043ce20c49a6)
There are a some instances where `0.0625rem` is used; still tho `eighth` is kind of small?

Reply:
very small... one pixel in fact :/ 

This seems to exist to vertically center the text with the radio button. there may be a way to do the same via flex and avoid this kinda thing altogether. So as not to over-complicate this PR, we can make a comment on why such a precise value is being used, then leave it be for now

Reply:
That makes sense to me, will do